File: abc010/src/b.cpp
#include <iostream>
#include <vector>

using namespace std;

int main()
{
    int n;
    cin >> n;
    vector<long long> a(n);
    for (int i = 0; i < n; i++)
    {
        cin >> a[i];
    }
    long long sum = 0;
    for (long long a_i : a)
    {
        long long m = a_i;
        while (m % 2 == 0 || m % 3 == 2)
        {
            m--;
            sum++;
        }
    }
    cout << sum << endl;
    return 0;
}

File: abc010/src/b.go
package main

import "fmt"

func main() {
	var n int
	fmt.Scan(&n)

	a := make([]int64, n)
	for i := 0; i < n; i++ {
		fmt.Scan(&a[i])
	}

	var sum int64 = 0
	for _, a_i := range a {
		m := a_i
		for m%2 == 0 || m%3 == 2 {
			m--
			sum++
		}
	}

	fmt.Println(sum)
}
